daemon/src/bitmex_price_feed.rs

```diff
@@ -67,6 +68,25 @@ impl Quote {
             ask: Usd::from(Decimal::try_from(quote.ask_price)?),
         }))
     }
+
+    #[allow(dead_code)] // Not used by all binaries.
+    pub fn for_maker(&self) -> Usd {
+        // XXX: is this the correct assumption?
+        self.ask
+    }
+
+    #[allow(dead_code)] // Not used by all binaries.
+    pub fn for_taker(&self) -> Usd {
+        // XXX: is this the correct assumption?
```

Comment:
I think it should be the `bid` price, if we assume an arbitrage model where the seller sells what he gets back on the market. The seller sells at the `bid` price, so the seller is covered with the `bid`. 
Note: In the end mid_range is fine too - we can always make this part of the interest calc (which is currently not implemented yet). 
Feel free to just go with mid_range for now. 
Please either convert the `XXX` to be a `TODO` (if we should track this further), or just remove the comment.

Reply:
I had some chats with @bonomat  about this, and we decided to go with mid_range for now. As I'm not sure whether this is correct, I'll convert it to a TODO.

Reply:
For the taker it is just "informative" and `mid_market` makes sense. For the `maker` it should be the `ask` price, because that's the same rate he would get on Bitmex if he'd hedge his position. 
Note: in our hedging model, the `maker` has to `buy` on Bitmex as he is `selling` on Hermes - he is literally mirroring the position.